exercises/sheet4/marta_bustins_marta_gonzalez_esteve_tarrago/4.tex

```diff
@@ -0,0 +1,186 @@
+
+  \bigskip\bigskip
+\item Show that
+  \[
+    \int_P f(x)\,\text{d}x
+    \ = \
+    vol(P) \cdot f(p_0)
+  \]
+  for any polytope $P$ and linear function $f$, where $p_0 = \frac{1}{vol(P)}\int_P x\,\text{d}x$ denotes the barycenter of $P$.
+  \bigskip
+  
+  Let $f$ be a linear function $f:\mathbb{R}^d\longrightarrow\mathbb{R}^n$.
+  Then $f$ can be expressed as $f(x)=Ax+b$ where $A$ is a $nxd$ matrix and $b$ is a $nx1$ vector. Then:
+  
+  \begin{equation*}
+       \int_P f(x)\,\text{d}x= \int_P Ax+b\,\text{d}x = A\int_P x\,\text{d}x + b\int_P \,\text{d}x =  (vol P) \bigg(\frac{A}{vol(P)}\int_P x\,\text{d}x + b\bigg) = vol(P)*f(p_0)
+  \end{equation*}
+  where $p_0 = \frac{1}{vol(P)}\int_P x\,\text{d}x$ is the barycenter of $P$.
+  \bigskip
+\item Complete the proof of Theorem 9.6 in Ziegler's \emph{Lectures on Polytopes}.
+  
+\end{enumerate}
+\begin{defi}
+Let $\gamma$ be a section. $\gamma$ is a tight section if and only if $\mathrm{Im}\gamma=\bigcup_{F\in\mathcal{F}}F$ for some $\mathcal{F}\subseteq\mathcal{L}\left(P\right)$.
+\end{defi}
+\begin{obs}
+If $\gamma$ is a section but it is not tight, then there exists $p\in\mathrm{Im}\gamma$ such that $p\not\in\bigcup_{F\in\mathcal{F}}F$ for any $\mathcal{F}\subseteq\left\lbrace\mathrm{proper\,faces\,of\,P}\right\rbrace$.
+\end{obs}
+
+%\textcolor{red}{crec que falta teorema al que es refereix la prova} \textcolor{blue}{A veure, aquí la intenció era posar el teorema TEOREMA, en plan abans de posar-lo primer fixem nocions i fem alguna observació,i després doncs TEOREMA: bla bla bla DEMOSTRACIÓ: bla bla bla perquè crec que quedaria una mica més endraçat... Però si teniu altres opcions endavant!}
+
+
+
+\textit{Proof}. Let $\gamma$ and $\rho$ be two sections of $P$, and let $\lambda\in\mathbb{R}$. Then $\lambda\gamma+\left(1-\lambda\right)\rho$ is continuous since it is a linear combination of two continuous functions, and 
+\begin{align*}
+    \pi\left(\lambda\gamma+\left(1-\lambda\right)\rho\right)\left(x\right)&=\pi\left(\lambda\gamma\left(x\right)+\left(1-\lambda\right)\rho\left(x\right)\right)\\
+    &=\lambda\pi\left(\gamma\left(x\right)\right)+\left(1-\lambda\right)\pi\left(\rho\left(x\right)\right)\quad\qquad\mathrm{Since\,}\pi\mathrm{\,is\,linear}\\
+    &=\lambda x+\left(1-\lambda\right)x\qquad\qquad\qquad\qquad\quad\;\mathrm{Since\,}\gamma\mathrm{\,and\,}\rho\mathrm{\,are\,sections}\\
+    &=x\\
+\end{align*}
+so any convex combination of sections is a section (seen just by 2 sections, but repeating this argument by induction can be easily proved). In particular, this shows that $\Sigma\left(P,Q\right)$ is convex, since for any $x$, $y\in\Sigma\left(P,Q\right)$, $x=\frac{1}{\mathrm{volQ}}\int_Q\gamma\left(s\right)ds$, $y=\frac{1}{\mathrm{volQ}}\int_Q\rho\left(s\right)ds$, $\gamma$, $\rho$ two sections and $\lambda\in\left[0,1\right]$ we get that
```

Comment:
you don't need the part in parentheses. But please start a new line for every sentence!